src/services/StationsService.ts

```diff
@@ -0,0 +1,15 @@
+const locationsURI = 'http://localhost:8080/locations';
+
+export async function getLocations() {
+  const res = await fetch(locationsURI);
+  if (res.ok) {
+    const locations = await res.json();
+    return locations;
+  }
+  console.log('Failed to fetch locations.');
+  return [];
+}
+
+export function getLocation(sensorCode, locations) {
+  return locations.find(val => val.code === sensorCode);
+}
```

Comment:
This is probably not a responsibility of the service. You declare this function in component that uses it or move it to `src/utils`. It is simple enough to just be used inline in the component's code. Also remember to specify types of properties.